config/jobs/kubernetes/sig-node/sig-node-presubmit-topologymanager.yaml

```diff
@@ -0,0 +1,120 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-node-e2e-topologymanager
+    optional: true
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-containerd
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+    always_run: false
+    max_concurrency: 12
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191212-28f7cbc-master
+        args:
+        - --root=/go/src
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
+        - --scenario=kubernetes_e2e
+        - -- # end bootstrap args, scenario args below
+        - --deployment=node
+        - --gcp-project=k8s-jkns-pr-node-e2e
+        - --gcp-zone=us-west1-b
+        - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
+        - --node-tests=true
+        - --provider=gce
+        - --test_args=--nodes=8 --focus="\[TopologyManager\]" --skip="" --flakeAttempts=2
```

Comment:
Can you point me to these tests, as I cannot find one? Or are you planning to create a new one?

Reply:
Also, will these run in normal CI (like the CPUManager tests https://testgrid.k8s.io/wg-resource-management#kubelet-serial-gce-e2e-cpu-manager), or will they just run as presubmit tests?

And, will they require more than the "baseline" single core gce instance?

Reply:
I have this PR for Topology Manager E2E tests. This is just the initial commit, will be more tests coming. For now, really wanted to run some (perhaps all?) of the CPU Manager tests against all of the policies supported by Topology Manager.

https://github.com/kubernetes/kubernetes/pull/86184


Ideally, we would like to have TopologyManager E2E tests run in normal CI, just like CPUManager, as well as presubmit tests, to validate PRs.

Good question about testing requirements. I'm thinking, it would be the same or very similiar to CPU Manager. We plan on adding some test cases for devices also (for example, SRIOV and GPU). The data will likely be "faked" from a real system.